js/form.js

```diff
@@ -0,0 +1,22 @@
+const addClass = (disabledItem) => {
+  disabledItem.classList.add('ad-form--disabled');
+};
+
+const addAtribute = (disabledItem) => {
```

Comment:
```suggestion
const addAttribute = (disabledItem) => {
```

надо добавить проверку орфографии в ide. Если не знаешь как - спроси меня